src/pcapng.h

```diff
@@ -153,20 +158,39 @@ PcapNgWriteEnhancedPacket(
     long InterfaceId,
     long IsSend,
     long TimeStampHigh, // usec (unless if_tsresol is used)
-    long TimeStampLow
+    long TimeStampLow,
+    ULONG ProcessID
     )
 {
     int Err = NO_ERROR;
     struct PCAPNG_BLOCK_HEAD Head;
     struct PCAPNG_ENHANCED_PACKET_BODY Body;
     struct PCAPNG_BLOCK_OPTION_ENDOFOPT EndOption;
     struct PCAPNG_BLOCK_OPTION_EPB_FLAGS EpbFlagsOption;
+    struct PCAPNG_BLOCK_OPTION_COMMENT CommentOption;
     struct PCAPNG_BLOCK_TAIL Tail;
     char Pad[4] = {0};
+// COMMENT_MAX_SIZE must be multiple of 4
+#define COMMENT_MAX_SIZE 16
+    char Comment[COMMENT_MAX_SIZE] = { 0 };
+    size_t CommentLength = 0;
     int FragPadLength = (4 - ((sizeof(Body) + FragLength) & 3)) & 3; // pad to 4 bytes per the spec.
     int TotalLength =
```

Comment:
Move this initialization down to where CommentOption.Length has been computed so it will all be in one place rather than the extra "+=" line.